hat/README.md

```diff
@@ -0,0 +1,167 @@
+# HAT – Hono Admin Tool [![GitHub release](https://img.shields.io/github/release/ctron/hat.svg)](https://github.com/ctron/hat/releases)
```

Comment:
Better remove the image here? It refers and links to the *ctron/hat* repository (and the version number is 0.7.1, not the 0.7.2 defined here).